manifests/init.pp

```diff
@@ -205,7 +205,9 @@
   Integer[0] $api_service_worker_timeout = 90,
   Hash[String[1], String[1]] $api_client_auth_cn_map = {},
 ) {
-  $settings_file = "${config_dir}/settings.py"
+  $settings_file               = "${config_dir}/settings.py"
+  $db_encrypted_fields_keyfile = "${config_dir}/db_encrypted_fields_key"
+  $db_encrypted_fields_key     = extlib::cache_data('pulpcore_cache_data', 'db_encrypted_fields_key', generate_fernet_key())
```

Comment:
This must be a parameter. If you have multiple machines behind a load balancer you want a way have them share the same key.

Reply:
Good catch. I've updated to make it a parameter with the type `Pattern[/\A([a-zA-Z]|\d|-|_){43}=\z/]`